### DISCUSSION TITLE
What about moon rotation?

### DISCUSSION THREAD

**Discussion by b0iq** [jasonsturges/lunarphase-js#21] [Ideas]
Hi dear, 

I'm just wondering if I want to get moon rotation from certain coordinates. For example: 
moon rotation from latitude and longitude is -55deg and at the same date but different coordinates the rotation should be different.
So, anyone got an idea of the implementation for achieving that?

THANK YOU ALL FOR THIS AMAZING LIBRARY

**Comment by jasonsturges** [comment]
@b0iq Hi, thanks - yes, I was considering the following in #9:
- Latitude and Longitude
- Angle
- Altitude of moon at sunset in degrees
- Direction of moon at sunset
- Direction - raising / falling

So, distance in #10 and illumination calculation in #12.

For example, at this moment from location (~41N, ~93W) at 10pm CST:
- Moon location: 30.6963° above the horizon
- Moon is setting
- Distance: 238316.30 miles
- Compass bearing: 230.6454°
- Illumination 50.47%
- Angle of moon: 102.8 degrees clockwise (257.2 degrees counterclockwise) from zenith.

![image](https://user-images.githubusercontent.com/1213591/177681733-b5112532-b59e-47d8-9b2a-358c76adf9f1.png)

Or, do you mean the synchronous rotation of the moon?

**Reply by b0iq** [reply]
@jasonsturges Thanks for your response, actually this is what I was seeking. I appreciate your work. 


I don't think I need the synchronous rotation of the moon. I just want to capture the moment of how the moon was looking like for certain location on earth and thats should be from getting the phase at some date and the angle at some coordinates. I'll wait for your PRs. 

Warm regards,
